pkg/secretless/entrypoint/entrypoint.go

```diff
@@ -60,6 +60,8 @@ func StartSecretless(params *SecretlessOptions) {
 		params.PluginChecksumsFile,
 		logger,
 	)
+	// Create the config validator
+	validateConfig := v2.NewValidateConfig(logger, availPlugins)
```

Comment:
why is the config validator so far from the point of use?

Reply:
I know you're not a fan of `-er` suffix but the current constructor name is really awkward - it's a double verb and it's almost implying that you are running the validation at this point rather than later.

Reply:
> why is the config validator so far from the point of use?

Thinking was to keep with the creation of the other dependencies.  Since it's essentially a derived dependency.  But I also considered putting it closer the point of use and since it was confusing to you I'll move it back there.